update.md

```diff
@@ -757,3 +757,61 @@ Le déploiement doit être autonome. Ce qui implique que :
 1. La mise à jour de dépendances est automatique et systématique,
 2. La personne qui déploie ne doit pas réfléchir (parce que c'est source d'erreur),
 3. La personne qui déploie ne doit pas avoir connaissance de ce qui est déployé (techniquement et fonctionnellement).
+
+
+ZEP-11
+------
+
+Les taches suivantes sont a executer après la MEP.
+
+### Modification du format de log nginx
+
+Ajouter la ligne ci-dessous dans le fichier `nginx.conf` pour modifier le format des logs
+
+
+    http {
+        log_format combined '$remote_addr - $remote_user [$time_local] "$request" $status $body_bytes_sent "$http_referer" "$http_user_agent" "$http_x_forwarded_for" $request_time $upstream_response_time $pipe';
+
+        ...
+    }
+
+Redemarrez ensuite le service nginx `service nginx restart`
+
+### Mise en place de rotation des logs
+
+Installer logrotate
+
+```bash
+sudo apt-get install logrotate
+```
+
+Si aucune rotation de log n'a été mise en place sur les access log nginx, il faudra créer le fichier ``/etc/logrotate.d/nginx.conf`` sinon, il suffira de modifier la configuration nginx existante.
+
+Le contenu doit être le suivant :
+
+```bash
+/var/log/nginx/access.log {
+    daily
+    rotate 90
+    compress
+    delaycompress
+    missingok
+    notifempty
+    create 644 zds zds
+}
+```
+
+Cette configuration permet d'effectuer une rotation tous les jours des access log de nginx, avec 90 jours de rétention. Les logs >= 2 jours seront compressés, ce qui permet au batch zds-stats de s'executer sur la log d'il y'a un jour et non compressée.
+
+Redemarrez ensuite le service logrotate `service logrotate restart`
+
+
+### Mise en place de l'ordonnancement de batchs via crontab
```

Comment:
```console
$ cat /etc/systemd/system/zds-index-solr.timer 
[Unit]
Description=Reindex SOLR Timer

[Timer]
OnCalendar=*:30:00
Persistent=true

[Install]
WantedBy=timers.target

$ cat /etc/systemd/system/zds-index-solr.service 
[Unit]
Description=Reindex SOLR Service

[Service]
Type=oneshot
User=zds
Group=zds
ExecStart=/opt/zds/zdsenv/bin/python /opt/zds/zds-site/manage.py index_content --only-flagged
ExecStart=/opt/zds/zdsenv/bin/python /opt/zds/zds-site/manage.py update_index --remove --age=1

```

Ce qui permet de faire par exemple :

```console
$ sudo systemctl status zds-index-solr.timer 
● zds-index-solr.timer - Reindex SOLR Timer
   Loaded: loaded (/etc/systemd/system/zds-index-solr.timer; enabled)
   Active: active (waiting) since mer. 2016-07-13 23:26:49 CEST; 1 months 9 days ago

Warning: Journal has been rotated since unit was started. Log output is incomplete or unavailable.
```